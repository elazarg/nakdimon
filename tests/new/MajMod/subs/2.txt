קֶפְּטֵן, אֵיפֹה אַתָּה? אֲנִי הוֹלֵךְ לְשֵׁם עִם הַמִּשְׁפָּחָה שֶׁל אֲחוֹתִי. אֲנַחְנוּ כִּמְעַט שָׁם, אָז חִכָּה. זוֹ הַדֶּרֶךְ הַנְּכוֹנָה? נָסַעְתִּי לְשֵׁם עֶשְׂרוֹת פְּעָמִים. כָּל שְׁאָר הַדְּרָכִים הָיוּ מְלֵאוֹת בִּמְכוֹנִיוֹת. אָנָּא תֵּן לָנוּ טְרֵמְפּ. יֵשׁ לִי יַלְדָּה קְטַנָּה. לִי יֵשׁ גַּם אִישָּׁה. אָנָּא קַח אוֹתָנוּ לַנָּמֵל. אֲנִי מִתְחַנֵּן בפניך... לֹא נִנְשַׁכְתִּי! זֶה לֹא נשיכה! אִם אֲנַחְנוּ נִרְאִים מְסוּכָּנִים, לְפָחוֹת קַח אֶת הַיַּלְדָּה שֶׁלָּנוּ. בְּבַקָּשָׁה, אֲדוֹנִי? בְּבַקָּשָׁה! אֲנִי מִתְחַנֶּנֶת בפניך! אֲדוֹנִי! בְּבַקָּשָׁה! יונג-סוק... אֶל תַּגִּידוּ כְּלוּם עַד שֶׁנִּהְיֶה בַּסְּפִינָה! אַתָּה יָכוֹל לוֹמַר לָנוּ מַדּוּעַ הַנְּגִיף הַנּוֹרָא הַזֶּה הִתְפַּשֵּׁט, פּוֹל? וּבְכֵן, לְמַרְבֵּה הַצַּעַר, אֵין לָנוּ מֵידָע רַב מִלְּבַד הָעוּבְדָּה שֶׁזֶּה הִתְחִיל ממפעל בִּיּוֹלוֹגִי בִּדְרוֹם קוֹרֵיאָה. אָה, לֹא בָּרוּר אֵיךְ הַנְּגִיף הַלֹּא מְזוֹהֶה הַזֶּה הִתְחִיל לְהִתְפַּשֵּׁט מַתְחִילָה. כַּמּוּבָן, תוהו וָבוֹהוּ מוּחְלָט הִתְרַחֵשׁ בִּדְרוֹם קוֹרֵיאָה וְהַרְבֵּה אֲנָשִׁים הֵחֵלּוּ לְהַאֲמִין לשמועה שהופצה בְּרֶשֶׁת... עַל איזור בָּטוּחַ אִי שָׁם דָּרוֹמָה. בוסאן? זוֹ ההגייה הַנְּכוֹנָה? זֶה נָכוֹן. אֲנָשִׁים פונו לְשֵׁם. אֲבָל בְּסוֹפוֹ שֶׁל דָּבָר שׁוּם מָקוֹם לֹא הָיָה בָּטוּחַ בֶּאֱמֶת. אֲבָל יֵשׁ מָקוֹם בָּטוּחַ בַּחֲצִי הָאִי הַקּוֹרֵיאָנִי. זֶה נָכוֹן. אֲנִי מַאֲמִין שֶׁאַתְּ מִתְכַּוֶּונֶת לְצָפוֹן קוֹרֵיאָה. אוּלַי זֶה מַזָּל שֶׁזֶּה קָרָה לִפְנֵי הָאִיחוּד! כָּכָה מִסְתַּבֵּר. אָז כַּמָּה זְמַן לִפְנֵי שממשלת קוֹרֵיאָה אִיבְּדָה אֶת הַשְּׁלִיטָה? רַק יוֹם אֶחָד.