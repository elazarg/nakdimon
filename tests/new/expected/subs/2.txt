קֶפְּטֵן, אֵיפֹה אַתָּה?
אֲנִי הוֹלֵךְ לְשָׁם עִם הַמִּשְׁפָּחָה שֶׁל אֲחוֹתִי.
אֲנַחְנוּ כִּמְעַט שָׁם, אָז חַכֵּה.
זוֹ הַדֶּרֶךְ הַנְּכוֹנָה?
נָסַעְתִּי לְשָׁם עֶשְׂרוֹת פְּעָמִים.
כָּל שְׁאָר הַדְּרָכִים הָיוּ מְלֵאוֹת בִּמְכוֹנִיּוֹת.
אָנָּא תֵּן לָנוּ טְרֵמְפּ.
יֵשׁ לִי יַלְדָּה קְטַנָּה.
לִי יֵשׁ גַּם אִישָּׁה.
אָנָּא קַח אוֹתָנוּ לַנָּמֵל.
אֲנִי מִתְחַנֵּן בְּפָנֶיךָ...
לֹא נִנְשַׁכְתִּי! זֶה לֹא נְשִׁיכָה!
אִם אֲנַחְנוּ נִרְאִים מְסוּכָּנִים, לְפָחוֹת קַח אֶת הַיַּלְדָּה שֶׁלָּנוּ. בְּבַקָּשָׁה, אֲדוֹנִי?
בְּבַקָּשָׁה! אֲנִי מִתְחַנֶּנֶת בְּפָנֶיךָ!
אֲדוֹנִי! בְּבַקָּשָׁה!
יוֹנְג-סוֹק...
אַל תַּגִּידוּ כְּלוּם עַד שֶׁנִּהְיֶה בַּסְּפִינָה!
אַתָּה יָכוֹל לוֹמַר לָנוּ מַדּוּעַ הַנְּגִיף הַנּוֹרָא הַזֶּה הִתְפַּשֵּׁט, פּוֹל?
וּבְכֵן, לְמַרְבֵּה הַצַּעַר, אֵין לָנוּ מֵידָע רַב מִלְּבַד הָעוּבְדָּה שֶׁזֶּה הִתְחִיל מִמִּפְעָל בִּיּוֹלוֹגִי בִּדְרוֹם קוֹרֵיאָה.
אָה, לֹא בָּרוּר אֵיךְ הַנְּגִיף הַלֹּא מְזוֹהֶה הַזֶּה הִתְחִיל לְהִתְפַּשֵּׁט מִתְּחִילָּה.
כַּמּוּבָן, תּוֹהוּ וָבוֹהוּ מוּחְלָט הִתְרַחֵשׁ בִּדְרוֹם קוֹרֵיאָה וְהַרְבֵּה אֲנָשִׁים הֵחֵלּוּ לְהַאֲמִין לִשְׁמוּעָה שֶׁהוּפְצָה בָּרֶשֶׁת... עַל אֵיזוֹר בָּטוּחַ אֵי שָׁם דָּרוֹמָה.
בּוֹסָאן? זוֹ הַהֲגִיָּיה הַנְּכוֹנָה?
זֶה נָכוֹן.
אֲנָשִׁים פּוּנּוּ לְשָׁם.
אֲבָל בְּסוֹפוֹ שֶׁל דָּבָר שׁוּם מָקוֹם לֹא הָיָה בָּטוּחַ בֶּאֱמֶת. אֲבָל יֵשׁ מָקוֹם בָּטוּחַ בַּחֲצִי הָאִי הַקּוֹרֵיאָנִי.
זֶה נָכוֹן.
אֲנִי מַאֲמִין שֶׁאַתְּ מִתְכַּוֶּונֶת לִצְפוֹן קוֹרֵיאָה.
אוּלַי זֶה מַזָּל שֶׁזֶּה קָרָה לִפְנֵי הָאִיחוּד!
כָּכָה מִסְתַּבֵּר.
אָז כַּמָּה זְמַן לִפְנֵי שֶׁמֶּמְשֶׁלֶת קוֹרֵיאָה אִיבְּדָה אֶת הַשְּׁלִיטָה?
רַק יוֹם אֶחָד.
